src/Tooltip/Tooltip.module.scss

```
@@ -1,17 +1,19 @@
 .tooltip {
   border-radius: 6px;
   padding: 15px;
- 
   background-color: var(--colour-surface-light);
   color: var(--colour-background-light);
-
   &--white {
     background-color: var(--colour-background-white);
     box-shadow: 0 1px 4px rgba(0, 0, 0, 0.3);
     color: var(--colour-text-900);
   }
 }
 
+.tooltipContainer {
+  z-index: var(--zindex-tooltip)
```

```suggestion
  z-index: var(--zindex-tooltip);
```
Remember to run `npm run prettier` to check for formatting fixes.